.github/workflows/push-deploy.yml

```
@@ -0,0 +1,19 @@
+name: Deploy to WordPress.org
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  tag:
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: pxgamer/composer-action@master
```

```suggestion
    - uses: shivammathur/setup-php@v2
      with:
            php-version: '7.3'
            coverage: none
            tools: composer
```

I'd switch to the `shivammathur/setup-php` action, development seems to be more active but also allows to define a PHP version.